horovod/spark/lightning/remote.py

```diff
@@ -75,46 +76,65 @@ def RemoteTrainer(estimator, metadata, ckpt_bytes, run_id, dataset_idx, train_ro
     # Storage
     store = estimator.getStore()
     remote_store = store.to_remote(run_id, dataset_idx)
-    is_dbfs = isinstance(store, DBFSLocalStore)
+    #is_dbfs = isinstance(store, DBFSLocalStore)
```

Comment:
If this is no longer needed, we should remove this line.

Reply:
removed.